css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Was this file not going to be a LESS file?  In which case the colours can be defined as variables, rather than having to repeat the value (and possibly get it slightly wrong each time).